ProcessMaker/processmaker
# FOUR-7955 | Implement the New Ellipsis Menu for Specified Menus Across the Platform

# Description
Ticket: [FOUR-7955](https://processmaker.atlassian.net/browse/FOUR-7955)

The new Ellipsis menu needs to be implemented across the platform for specific menus, which will be provided in a list. 
The top three menus that require the new Ellipse menu are:

- 	Tasks/To Do Tasks
- 	Designer/Scripts
- 	Designer/Screens

# How to Test
1. Go to branch `feature/FOUR-7955` in core.
2. Go to Tasks -> To Do Tasks
	- View Ellipsis Menu and test functionality (linking)
3. Go to Designer -> Scripts
	- View Ellipsis Menu and test functionality (linking)
	- Also check that you can click on the Script name
4. Go to Designer -> Screens
	- View Ellipsis Menu and test functionality (linking)
	- Also check that you can click on the Screen name

- [FOUR-7955](https://processmaker.atlassian.net/browse/FOUR-7955) | Implement the New Ellipsis Menu for Specified Menus Across the Platform
- [FOUR-7878](https://processmaker.atlassian.net/browse/FOUR-7637) | Ellipsis Menu

# Code Review Checklist


[FOUR-7955]: https://processmaker.atlassian.net/browse/FOUR-7955?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-7955]: https://processmaker.atlassian.net/browse/FOUR-7955?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-7878]: https://processmaker.atlassian.net/browse/FOUR-7878?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
